### DISCUSSION TITLE
It is desirable to obtain methods to generate annotation files for virulence factors and resistance genes separately.

### DISCUSSION THREAD

**Discussion by fersity** [oschwengers/bakta#351] [General]
Hello, excuse me, I think you did a great job of bringing together annotation results from multiple important databases. But sometimes I would like to make it easier to single out interesting database annotation results for subsequent analysis and visualization. Is there a way to generate separate annotation files for VFDB, CARD, AMRFinderPlus and other annotated genes and transposons? thank you.

**Comment by oschwengers** [comment]
Hi thanks for reaching out with this. This is an interesting question. Unfortunately, during the creation of the Bakta db we do not store all annotation from the varying source next to each other, but instead overwrite existing with more specific annotations. However, at runtime, Bakta does store different annotation information like for example from AMRFinderplus or PSCs pre-annotated with information from VFDB. So, for AMRFinder it is fairly easy to extract such information. But as far as I understood this correctly, you would like to extract the different information for a given protein from different annotation databases. This is not passible due to the underlying structure of the Bakta database. Otherwise, this would imply a huge increase in storage requirements, which is already fairly high.
